vcd/datasource_vcd_vapp_vm.go

```diff
@@ -0,0 +1,137 @@
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"vdc": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The name of VDC to use, optional if defined at provider level",
+			},
+			"computer_name": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Computer name assigned to this virtual machine",
+			},
+			"description": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The VM description",
+				// Currently, this field has the description of the OVA used to create the VM
+			},
+			"memory": &schema.Schema{
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "The amount of RAM (in MB) to allocate to the VM",
+			},
+			"cpus": &schema.Schema{
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "The number of virtual CPUs to allocate to the VM",
+			},
+			"cpu_cores": &schema.Schema{
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "The number of cores per socket",
+			},
+			"metadata": {
+				Type:        schema.TypeMap,
+				Computed:    true,
+				Description: "Key value map of metadata to assign to this VM",
+			},
+			"href": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "VM Hyper Reference",
+			},
+			"storage_profile": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Storage profile used with the VM",
+			},
+			"network": {
+				Computed:    true,
+				Type:        schema.TypeList,
+				Description: "",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "Network type",
+						},
+						"ip_allocation_mode": {
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "IP address allocation mode.",
+						},
+						"name": {
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "Name of the network this VM should connect to.",
+						},
+						"ip": {
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "IP of the VM. Settings depend on `ip_allocation_mode`",
+						},
+						"is_primary": {
+							Computed:    true,
+							Type:        schema.TypeBool,
+							Description: "Set to true if network interface should be primary. First network card in the list will be primary by default",
+						},
+						"mac": {
+							Computed:    true,
+							Type:        schema.TypeString,
+							Description: "Mac address of network interface",
+						},
+					},
+				},
+			},
+			"disk": {
+				Type: schema.TypeSet,
+				Elem: &schema.Resource{Schema: map[string]*schema.Schema{
+					"name": {
```

Comment:
Why don't we expose `bus_number` and `unit_number`? These two fields would be potentially pretty useful.

Reply:
They are not available after VM creation

Reply:
They are if you look at VM details in UI:

<img width="1470" alt="Screen Shot 2019-11-06 at 15 30 14" src="https://user-images.githubusercontent.com/11330689/68302293-5f357280-00aa-11ea-92bf-aedc90456c57.png">

Reply:
<img width="992" alt="Screenshot 2019-11-06 14 35 47" src="https://user-images.githubusercontent.com/11331818/68303277-d4517980-00a3-11ea-8f33-c637bd1ea73d.png">

I trusted the opinion of the previous implementer about this matter.
Such information is not in the VM structure. I need to find out where it is is stored.

Reply:
Found it. It is stored in a structure that is returned using API 32+.
([VmSpecSectionType](https://vdc-download.vmware.com/vmwb-repository/dcr-public/037ccaee-649a-417e-b365-1331034fb28d/1f0fd9eb-0238-4af6-89b5-7e6636f29c65/doc/doc/types/VmSpecSectionType.html))
We can't use it currently, as we are still bound to version 27